### ISSUE TITLE
webots.min.js: ANSI escape code not processed when printing on the console

### ISSUE THREAD

**Issue by stefaniapedrazzi** [cyberbotics/webots#818]
ANSI escape codes are handled in the Webots console but not in the webots.min.js console.

**Steps to Reproduce**
1. Stream the `camera.wbt`
2. Connect with the `streaming_viewer/index.html`
3. Check the text printed in the console

Instead of changing the color of the text, the ANSI escape codes are written as text in the console
```
[camera] Looks like I found a [32mgreen[0m blob.

[camera] Looks like I found a [34mblue[0m blob.
```

**Comment by stefaniapedrazzi** [comment]
Currently, there is no console in the in the web interface.
Not sure when and if it is planned to add it back.
So I will postpone this because it is something we would like to test when a console is available.

However, I checked in the browser console:
* in Chrome it works correctly
* in Firefox the ANSI escape code are not processed

**Comment by stefaniapedrazzi** [comment]
Fixed in  #5119.
